pkg/operator/schema/schema.go

```diff
@@ -52,7 +52,8 @@ type DeployResult struct {
 }
 
 type GetAPIsResponse struct {
-	SyncAPIs []SyncAPI `json:"sync_apis"`
+	SyncAPIs    []SyncAPI     `json:"sync_apis"`
+	APISplitter []APISplitter `json:"api_splitter"`
```

Comment:
APISplitter -> APISplitters and `json:"api_splitter"` -> `json:"api_splitters"` because this is a list of APISplitters